documentation/modules/proc-installing-using-template.adoc

```diff
@@ -20,12 +20,12 @@ oc login -u system:admin
 +
 [options="nowrap",subs="attributes"]
 ----
-oc new-project enmasse
+oc new-project {ProductNamespace}
 ----
 
 . Deploy using `enmasse-with-standard-authservice` template
 +
 [options="nowrap",subs="attributes"]
 ----
-oc process -f install/templates/enmasse-with-standard-authservice.yaml NAMESPACE=enmasse | oc apply -f -
+oc process -f install/templates/enmasse-with-standard-authservice.yaml NAMESPACE={ProductNamespace} | oc apply -f -
 ----
```

Comment:
<{ProductNamespace}>

Immediately after the codeblock add:
"where <{ProductNamespace}> is the namespace where {ProductName} is deployed."